docs/datatype/image/text2image.md

```diff
@@ -1,3 +1,484 @@
 # Search Image from Text via CLIP model
 
-TBA
\ No newline at end of file
+In this tutorial, we will create an image search system that retrieves images based on short text descriptions of their content.
+
+The interest behind this is that regular search or queries rely on image description or meta data to describe the content of the image, this is simply not efficient 
+and expensive because you need a person to write that description and also information about image content is not always available.
+
+We need to look for another solution ! What if we directly compare text with images ?  
+
+To do so, we need to figure out a way to match images and texts. One way to find such matches we need to find related images with similar semantics to the query texts. This requires us to represent both images and query texts in the same embedding space to be able to do the matching. In this case, pre-trained cross-modal models can help us out.
```

Comment:
- match images and text (no s)
- One way to find such matches we need to find... -> One way to find such matches is finding...
- "texts" -> "text" (all occurrences in the doc)